external/CMakeLists.txt

```diff
@@ -290,7 +290,7 @@ function(add_repo_tests REPO)
 
     add_test(
       NAME "${TEST_EXPECTATION_NAME}"
-      COMMAND $<TARGET_FILE:toit.run> "${FILE}" $<TARGET_FILE:toit.run>
+      COMMAND $<TARGET_FILE:toit.run> "${FILE}" $<TARGET_FILE:toit>
```

Comment:
So we still use `toit.run` for running the test, but pass `toit` to it?

Reply:
Correct.
We could use `toit` for running the tests as well, but inside the Toit repository I'm fine with using the individual tools.